webapp/src/package.ts

```diff
@@ -15,8 +16,8 @@ let extWeight: pxt.Map<number> = {
 }
 
 export function setupAppTarget(trgbundle: pxt.TargetBundle) {
-    //if (!trgbundle.appTheme) trgbundle.appTheme = {};
     pxt.setAppTarget(trgbundle)
+    compiler.initAppTarget(trgbundle)
```

Comment:
the app target is modified when a different board variant is loaded so you probably need this in more places.

Reply:
Thanks, I now also set the app target when the appTargetVariant changes